examples/react-example/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "react-example",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@datapunt/matomo-tracker-react": "file:../../packages/react",
```

Comment:
Should we not simply provide the latest version here so it's easier to use for people getting started? During development we can always link this.

Reply:
not sure, in my case I added the example to see if nothing really broke after updating the dependencies (of course we have unit tests, but I wanted to see it actually work as well). I think the example could also be useful when developing